modules/rostests/apitests/ntdll/RtlBitmap.c

```diff
@@ -580,30 +580,43 @@ Test_RtlFindNextForwardRunClear(void)
 void
 Test_RtlFindFirstRunClear(void)
 {
+  todo_if(TRUE)
+    ok(FALSE, "%s() is UNIMPLEMENTED\n", __FUNCTION__);
 }
 
 void
 Test_RtlFindLastBackwardRunClear(void)
 {
+  todo_if(TRUE)
+    ok(FALSE, "%s() is UNIMPLEMENTED\n", __FUNCTION__);
 }
 
 void
 Test_RtlFindClearRuns(void)
 {
+  todo_if(TRUE)
+    ok(FALSE, "%s() is UNIMPLEMENTED\n", __FUNCTION__);
 }
 
 void
 Test_RtlFindLongestRunClear(void)
 {
+  todo_if(TRUE)
+    ok(FALSE, "%s() is UNIMPLEMENTED\n", __FUNCTION__);
 }
 
-
-START_TEST(RtlBitmap)
+START_TEST(RtlBitmapApi)
```

Comment:
As you're changing the test name entry point code, as a general rule, the file name must have the same name too.

Reply:
why is this renamed?